articles/troubleshooting/tsconfigtypewarning.md

```diff
@@ -0,0 +1,9 @@
+# The `tsconfig.json` Build Action Warning
+
+You may get an Intellisense warning that reads "One or more tsconfig.json files in this project have their Build Action set to None and as a result may not be correctly used for configuration." as shown below.
```

Comment:
I think it's usually written as "IntelliSense".

Reply:
Fixed.